protobuf/src/lib.rs

```diff
@@ -1,3 +1,5 @@
+extern crate alloc;
```

Comment:
Why is this necessary?

Reply:
i think when you turn on `no_std` you must put `extern crate alloc;` to get `alloc::vec::Vec` etc., or else you will link only to libcore. `no_std` without `extern crate alloc` is used for e.g. microcontrollers

Reply:
I'm sorry, this one (in protobuf crate) is not necessary for sure, fixing

Reply:
welp, I was wrong, I get a ton of build failures if I don't do this:

```
error[E0433]: failed to resolve: could not find `alloc` in `{{root}}`
   --> /home/chris/prost/prost/target/debug/build/protobuf-0d3e1b26d91b17a5/out/protobuf_unittest.rs:911:30
    |
911 |     pub unpacked_sfixed64: ::alloc::vec::Vec<i64>,
    |                              ^^^^^ could not find `alloc` in `{{root}}`

error[E0433]: failed to resolve: could not find `alloc` in `{{root}}`
   --> /home/chris/prost/prost/target/debug/build/protobuf-0d3e1b26d91b17a5/out/protobuf_unittest.rs:913:27
    |
913 |     pub unpacked_float: ::alloc::vec::Vec<f32>,
    |                           ^^^^^ could not find `alloc` in `{{root}}`
```